dutree/dutree.py

```diff
@@ -304,10 +305,38 @@ class DuScan:
     "Disk Usage Tree scanner"
 
     def __init__(self, pathname):
+        self._dev_allow = []
+        self._dev_deny = []
+        self._dev = lstat(pathname).st_dev
         self._path = self._normpath(pathname)
         self._tree = None
         self._check_path()
 
+    def skip_other_filesystems(self):
+        "add self._dev to self._dev_allow"
```

Comment:
This comment says exactly what you can read it does. So it is not helpful.

If you want a comment, then make it count:
```
"""
Do not traverse into other filesystems when scanning

This makes sense if you're looking to free space on a particular
filesystem.
"""
```